jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  setupFiles: ["<rootDir>/jest.setup.js"],
+  testPathIgnorePatterns: ["<rootDir>/.next/", "<rootDir>/node_modules/"]
```

Comment:
I am not sure but, shouldn't `<rootDir>/.next/` be `<rootDir>/client/.next/` ?

Reply:
Good catch! I've updated it to show the correct path `<rootDir>/client/.next/`